src/dashboards/Compact/index.tsx

```diff
@@ -68,22 +68,21 @@ const EnturDashboard = ({ history }: Props): JSX.Element => {
 
     const anyScooters = Boolean(scooters && scooters.length)
 
-    const localStorageLayout: Layouts =
-        getFromLocalStorage(history.location.key) || {}
-
     const bikeCol = anyBikeRentalStations ? 1 : 0
 
     const scooterCol = anyScooters ? 1 : 0
 
-    const cols = {
-        lg: numberOfStopPlaces + bikeCol + scooterCol,
-        md: numberOfStopPlaces + bikeCol + scooterCol,
+    const totalItems = numberOfStopPlaces + bikeCol + scooterCol
+
+    const cols: { [key: string]: number } = {
+        lg: totalItems >= 4 ? 4 : totalItems,
```

Comment:
Her går det an å bruke `Math.min(4, totalItems)`, men det er smak og behag.

Reply:
Ser mye ryddigere ut med ```Math.min()```, så endrer til det😄 Takk for innspill!